packages/h5wasm/src/guards.ts

```diff
@@ -61,6 +61,10 @@ export function isStringMetadata(metadata: Metadata) {
   return metadata.type === 3;
 }
 
+export function isArrayMetadata(metadata: Metadata) {
+  return metadata.type === 10;
+}
```

Comment:
Do you think we could use the `H5T_class_t` directly from `h5wasm` ?

Reply:
Not in its current state, since `H5T_class_t` is an `interface`, not an `enum`. I'll write it down in case I find time to open a PR on the h5wasm repo. :+1:

Reply:
Ah shoot, indeed. 

> I'll write it down in case I find time to open a PR on the h5wasm repo.

Thanks